certified-connectors/RecordedFutureV2/apiDefinition.swagger.json

```diff
@@ -1212,44 +1211,44 @@
                 "created_from_relative": {
                   "type": "string",
                   "enum": [
-                    "-2",
-                    "-24",
-                    "-48"
+                    "-5m",
+                    "-2h",
+                    "-1d"
                   ],
-                  "description": "Limit the response to playbook alerts created at most this many hours in the past. Defaults to '-168' (one week back).",
+                  "description": "Limit the response to playbook alerts created at most this many minutes, hours or days ago. Defaults to all time.",
                   "title": "Relative created from",
                   "x-ms-visibility": "advanced"
                 },
                 "created_until_relative": {
                   "type": "string",
                   "enum": [
-                    "-0",
-                    "-12",
-                    "-24"
+                    "-0m",
+                    "-2h",
+                    "-1d"
                   ],
-                  "description": "Limit the response to playbook alerts created at the latest this many hours in the past. Defaults to '0' (now).",
+                  "description": "Limit the response to playbook alerts created at the latest this many minutes, hours or days ago. Defaults to '-0' (now).",
                   "title": "Relative created until",
                   "x-ms-visibility": "advanced"
                 },
                 "updated_from_relative": {
                   "type": "string",
                   "enum": [
-                    "-2",
-                    "-24",
-                    "-48"
```

Comment:
@johannesh-recordedfuture An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it.
can you please confirm that no users will be affected by this change?

Reply:
@Amjed-Ayoub I can confirm, this will replace the previous version with the removed enum values and the previous version has not been certified yet.